daemon/api.go

```diff
@@ -956,6 +965,119 @@ func deleteCapability(c *Command, r *http.Request) Response {
 	}
 }
 
+// SkillGrant is a part of response from getSkills().
```

Comment:
Saying that foo exists because it's returned from bar isn't really helpful. Becomes much worse when foo is public and bar is private.